app/models/hearing.rb

```diff
@@ -30,8 +30,6 @@ class Hearing < ApplicationRecord
   validates :jurisdictionType, presence: true, inclusion: JURISDICTION_TYPES
   validates :court_centre_id, presence: true
   validates :hearing_type, presence: true
-  validates :hearing_days, presence: true
-  validates :hearingLanguage, inclusion: LANGUAGES
```

Comment:
Add
defendantJudicialResults
companyRepresentatives
isVacatedTrial
approvalRequested
seedingHearing
isVirtualBoxHearing
isSJPHearing

Reply:
To keep things more straightforward I've not added all the non-required fields. If needed they can be added in the future.